src/managers/note_manager.py

```diff
@@ -9,13 +9,34 @@
 class NoteManager:
     def __init__(self) -> None:
         self.notes: List[Note] = []
+ 
+    def validate_note(self, note: Note) -> bool:
+        """
+        Validates the note's title and content
 
+        """
+        if not note.title or len(note.title) < 5: #Number 5 is for example. Could be any number
+            print("Error: The title must not be empty and should have at least 5 characters.")
+            return False
+        if not note.content:
+            print("Error: The content must not be empty.")
+            return False
+        return True
```

Comment:
Hi @AndreyShvachko !
Please add an empty line between the methods.

Reply:
Hi @AlexnderDoroshenko !
Done! Thank you for your note!

Reply:
@AndreyShvachko hello!
How do you feel about us improving the logic a bit and making it more functional?
For example: Let's put a character limit in the argument with a default value:
def validate_note(self, note: Note, min_title_length: int = 5) -> None:
   if not note.title or len(note.title) < min_title_length:

Reply:
@MHryhoriev Hi!
Thanks for the note. Will do it